pallets/creditcoin/src/lib.rs

```diff
@@ -35,11 +35,22 @@ pub mod ocw;
 mod types;
 
 use ocw::tasks::collect_coins::GCreContract;
-pub use types::*;
+pub use types::{
+	loan_terms, Address, AddressId, AskOrder, AskOrderId, AskTerms, BidOrder, BidOrderId, BidTerms,
+	Blockchain, CollectedCoinsId, CollectedCoinsStruct, DealOrder, DealOrderId, Duration,
+	ExternalAddress, ExternalAmount, ExternalTxId, Guid, InterestRate, InterestType, LegacySighash,
+	LoanTerms, Offer, OfferId, OrderId, RatePerPeriod, Task, TaskId, TaskOutput, Transfer,
+	TransferId, TransferKind, UnverifiedCollectedCoins, UnverifiedTransfer,
+};
+
+pub(crate) use types::{DoubleMapExt, Id};
+
+#[cfg(test)]
+pub(crate) use types::test;
 
 pub type BalanceFor<T> = <T as pallet_balances::Config>::Balance;
 
-pub const STORAGE_VERSION: StorageVersion = StorageVersion::new(6);
+pub const STORAGE_VERSION: StorageVersion = StorageVersion::new(5);
```

Comment:
Hmmm, not sure why this has changed. Should it be reverted back to `6` now that migrations v6 is a noop? 

Alternatively should this match the version coming from migrations and actually be 8 instead ?

Reply:
Ah, good catch! It should be 8. We must've forgotten to bump this when we added v7 and v8...

Reply:
Opened https://github.com/gluwa/creditcoin/pull/1027 with an extra check for this.